apiary.apib

```diff
@@ -5747,12 +5749,12 @@ Use this API call to obtain the definition of component available in KBC.
                 "uri": "https://syrup.keboola.com/docker/keboola.app-tree",
             }
 
-### Get development branch component [GET /v2/storage/branch/{branch_id}/components/{componentId}]
+### Get component (deprecated) [GET]
 
-Use this API call to obtain the definition of component for [Development Branch](#reference/development-branches) available in KBC.
+***Deprecated**, use method listed above.*
+Use this API call to obtain the definition of component available in KBC.
```

Comment:
To bych řekl, že není dobrý způsob. Mělo by to být zalinkované. Třeba při migraci na OpenAPI už nemusí být "above"